assets/css/00-settings/_colors.scss

```diff
@@ -1,31 +1,38 @@
-// Brand/Base colors
-$color-primary: #f0f;
-$color-secondary: #ff0;
-$color-tertiary: #0ff;
-
-// Status colors
-$color-red: #fc3030;
-$color-green: #74bd00;
-
-// Grays
-$color-gray-light: #f5f5f6;
-$color-gray-medium: #e0e0e4;
-$color-gray-dark: #2c2f33;
-
-// Dark and light
-$color-dark: #000;
-$color-light: #fff;
-
-// Font Colors
-$color-base-font: $color-gray-dark;
-$color-action: $color-primary;
-
-// Button Colors
-$color-base-button: $color-primary;
-$color-secondary-button: $color-secondary;
-
-// Border Colors
-$color-base-border: $color-gray-medium;
-
-// Background Colors
-$color-base-background: $color-light;
+/*  =========================================================================
+    settings.colors
+    Project wide color settings.
+    ========================================================================= */
+
+// Brand/Base colors
+$color-primary: #f0f !default;
+$color-secondary: #ff0 !default;
+$color-tertiary: #0ff !default;
+
+// Status colors
+$color-error: #fc3030 !default;
+$color-success: #74bd00 !default;
+$color-warning: #f8d407 !default;
+
+// Grays
+$color-gray-light: #f5f5f6 !default;
+$color-gray-medium: #cccccc !default;
+$color-gray-dark: #2c2f33 !default;
+
+// Dark and light
+$color-dark: #000 !default;
+$color-light: #fff !default;
+
+// ^refactor: do we need this?
```

Comment:
@iainvdw ff checken of we hier nog iets van nodig hebben

Reply:
Dark en light kunnen we wel houden, mocht het thema geen zwart en wit als lichte en donkere kleuren hebben.